handlers/api.go

```diff
@@ -302,17 +313,21 @@ func ApiEpoch(w http.ResponseWriter, r *http.Request) {
 		(SELECT COUNT(*) FROM blocks WHERE epoch = $1 AND status = '0') as scheduledblocks,
 		(SELECT COUNT(*) FROM blocks WHERE epoch = $1 AND status = '1') as proposedblocks,
 		(SELECT COUNT(*) FROM blocks WHERE epoch = $1 AND status = '2') as missedblocks,
-		(SELECT COUNT(*) FROM blocks WHERE epoch = $1 AND status = '3') as orphanedblocks,
-		(SELECT to_timestamp($2)) as ts
-		FROM epochs WHERE epoch = $1`, epoch, utils.EpochToTime(uint64(epoch)).Unix())
+		(SELECT COUNT(*) FROM blocks WHERE epoch = $1 AND status = '3') as orphanedblocks
+		FROM epochs WHERE epoch = $1`, epoch)
 	if err != nil {
 		logger.WithError(err).Error("error retrieving epoch data")
 		sendServerErrorResponse(w, r.URL.String(), "could not retrieve db results")
 		return
 	}
 	defer rows.Close()
 
-	returnQueryResults(rows, w, r)
+	epochTime := func(dataEntryMap map[string]interface{}) error {
```

Comment:
**Minor:** Personally I would name this `addEpochTime`.

IF you change that, keep in mind that the "add" should be added here too:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/817b4c30dfccb52a0821a6106467af78cd5e0766/handlers/api.go#L254
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/817b4c30dfccb52a0821a6106467af78cd5e0766/handlers/api.go#L1573